packages/flutter_tools/test/general.shard/web/devfs_web_ddc_modules_test.dart

```diff
@@ -0,0 +1,1363 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I'm guessing this is copy paste?

Reply:
Yep - updated!

Reply:
The linter complains if this is anything besides '2014'. Is this intended?

Reply:
Sorry, miscommunication. 2014 IS correct, the copyright for a "piece" of software (in this case the Flutter SDK) should be the FIRST year it was copyrighted.

I was merely verifying that this huge test library that I don't plan to review was mostly a copy paste of an existing test, with a few updates.

Reply:
Ah, gotcha. You're all good then!